.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request: {}
+
+jobs:
+  test:
+    name: test node v${{ matrix.node }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [12, 16]
```

Comment:
in the future when we have tests, we should probably test agsinst both — for now this can also just be 16 or just 12

```suggestion
        node: [16]
```

```suggestion
        node: [12]
```